docs/api/app.md

```diff
@@ -614,6 +614,8 @@ Returns `String` - The current application directory.
 Returns `String` - A path to a special directory or file associated with `name`. On
 failure, an `Error` is thrown.
 
+Calling `app.getPath('logs')` without having previously called `app.setAppLogsPath()` will result in a directory being created and set to `/Library/Logs/YourAppName` on _macOS_, and inside the `userData` directory on _Linux_ and _Windows_.
```

Comment:
```suggestion
If `app.getPath('logs')` is called without called `app.setAppLogsPath()` being called first, a default log directory will be used equivalent to calling `app.setAppLogsPath()` without a `path` parameter.
```

(point back to the description in `app.setAppLogsPath()` instead of re-describing it here)

Reply:
Unresolving due to lack of response + lack of change?

Reply:
whoops, thought this made it into the changes and did not. resolved.